Floofbot/Services/NicknameAlertService.cs

```diff
@@ -64,6 +64,9 @@ public async Task OnReactionAdded(Cacheable<IUserMessage, ulong> message, ISocke
         {
             var msg = message.Value as IUserMessage;
 
+            if (reaction.User.Value.IsBot)
```

Comment:
Is this to prevent bots from calling this event handler?

Reply:
yes, the bot can trigger its own event handler